src/rebar_prv_shell.erl

```diff
@@ -442,16 +445,24 @@ reread_config(AppsToStart, State) ->
             ok
     end.
 
-boot_apps(Apps) ->
+boot_apps(Apps, BootMessageMode) ->
     Normalized = normalize_boot_apps(Apps),
     Res = [application:ensure_all_started(App) || App <- Normalized],
-    _ = [?INFO("Booted ~p", [App])
-            || {ok, Booted} <- Res,
-            App <- Booted],
+    print_booted([App || {ok, Booted} <- Res, App <- Booted], BootMessageMode),
```

Comment:
One of the thing we got with the current implementation is help when apps may get stuck or stalled in case of issues getting started. Otherwise if you have 12 apps to load and the 9th gets stuck, you never have any idea what is going on.

This is unfortunately not super compatible with the short format if there's stuff being printed during app boots however.

Reply:
There is undocumented `application:info` to see what is the current state, and why not all applications are booting correctly.